src/rpc/server.cpp

```diff
@@ -322,6 +322,7 @@ void StopRPC()
 {
     LogPrint("rpc", "Stopping RPC\n");
     deadlineTimers.clear();
+    DeleteAuthCookie();
```

Comment:
At least it's a separate commit in this PR. But splitting it to a separate PR would make sense, yes.